bottlecap/src/config/yaml.rs

```diff
@@ -91,6 +93,13 @@ pub struct YamlConfig {
     pub capture_lambda_payload_max_depth: Option<u32>,
     #[serde(deserialize_with = "deserialize_optional_bool_from_anything")]
     pub serverless_appsec_enabled: Option<bool>,
+    pub appsec_rules: Option<String>,
+    #[serde(default)]
```

Comment:
Do you really need the `#[serde(default)]`? The `YamlConfig` field already includes it, which should be passed down to this fields

Reply:
Ah you're right I forgot this could also be set at container level!